keyboards/bm16s/readme.md

```diff
@@ -0,0 +1,13 @@
+# bm16s
+
+A 16 key macropad, with USB C, RGB underglow and backlight, it's a variant of the BM16A, but with low profile cioc switches.
```

Comment:
Does this PCB actually support underglow? I can't see any pads for it in the photos on Massdrop. All the LEDs I see are the "in-switch" RGBs.

Reply:
They're PCB mounted LED's similar to how the ALT and CTRL are done.
![IMG_6733](https://user-images.githubusercontent.com/127934/56454137-3e215600-6301-11e9-87c1-afe6fb8dce37.jpeg)

Reply:
That's in-switch though. Underglow would face the bottom of the PCB and illuminate the case (if it were plastic).

Reply:
RGB Underglow is usually denoted to RGB Light, with strips under the PCB, facing down.   
https://docs.qmk.fm/#/feature_rgblight

While RGB Matrix is per key RGB LEDs, which is what you have here. 
https://docs.qmk.fm/#/feature_rgb_matrix


So, if you could please change this, so it may be less confusing for people? (aka, we would like to keep the wording consistent here).